ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/services/blob-video-storage.service.spec.ts

```diff
@@ -1,30 +1,35 @@
 import { BlobVideoStorageService } from './blob-video-storage.service';
 import { BlobStorageService } from './blob-storage.service';
+import { DeviceType } from './device-type';
 import { Config } from '../../shared/models/config';
+import { VideoFiles } from './video-files';
 
 describe('BlobVideoStorageService', () => {
```

Comment:
What happens if there is an enumeration value (VideoFiles.x) for which there isn't a mapped resolution? Do we throw an error?

Reply:
you can not pass value that is not exists(as it should be emun), could be that file does not exist, this error is cached by UI

Reply:
What I mean is, it is possible that we add another value to the enumeration but forget to add a mapping. In that case, someone might believe it is working but in fact, it will fail returning `undefined`. It's better to catch this in the class and return an error.

Reply:
it's done